modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/planner/hints/ForceIndexHintPlannerTest.java

```diff
@@ -88,7 +90,6 @@ public void testSingleTable() throws Exception {
     public void testSubquery() throws Exception {
         var sql1 = "SELECT /*+ FORCE_INDEX({}) */ * FROM TBL1 t2, (SELECT * FROM TBL1 WHERE val2='v' AND val3='v') t1 WHERE t2.val2='v'";
 
-        assertPlan(format(sql1, ""), SCHEMA, nodeOrAnyChild(isTableScan(TBL1)));
```

Comment:
Does it mean we shouldn't see a plan with table scan?
`assertPlan(format(sql1, ""), SCHEMA, 
   not(nodeOrAnyChild(isTableScan(TBL1)));`

Reply:
It depends from planer configuration and row count, so I trying to avoid of this.